app/src/main/aidl/com/snaggly/ksw_toolkit/IKSWToolKitService.aidl

```diff
@@ -48,6 +48,9 @@ interface IKSWToolKitService {
 
     boolean getNightBrightness();
     void setNightBrightness(boolean value);
-
+    
+    boolean gethideStartMessage();
+    void sethideStartMessage(boolean value);
+    
```

Comment:
Consider that this might change if you change the order from KSWToolKit-Service.

Reply:
I don't really understand it, but I made it last anyway, as well as the service